clients/Application/cypress/e2e/1-getting-started/basic-access.cy.js

```diff
@@ -0,0 +1,85 @@
+/// <reference types="cypress" />
```

Comment:
Can you please add a readme under cypress folder on how to run these tests.

Reply:
Add info on running in headless and headed mode.

Reply:
added a README under the cypress folder